modules/serverless-broker-ingress-metrics.adoc

```diff
@@ -0,0 +1,25 @@
+[id="serverless-broker-ingress-metrics_{context}"]
+= Broker ingress metrics
+
+You can use the following metrics to debug the broker ingress, see how it is performing, and see which events are being dispatched by the ingress component.
```

Comment:
It would be strange if we capitalize Ingress but not Broker, or any other API objects, without using the API formatting like this: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references

I think since it's a [general reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references) let's leave it lowercase for now?